src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java

```diff
@@ -107,6 +107,10 @@ public String getSubmitActionQuestion() {
         return isModeration ? Const.ActionURIs.INSTRUCTOR_EDIT_STUDENT_FEEDBACK_SAVE
                               : Const.ActionURIs.STUDENT_FEEDBACK_QUESTION_SUBMISSION_EDIT_SAVE;
     }
+
+    public String getInstructorSubmitActionQuestion() {
+        return Const.ActionURIs.INSTRUCTOR_FEEDBACK_QUESTION_SUBMISSION_EDIT_SAVE;
+    }
```

Comment:
Nope, I saw that it was being used this way in the studentFeedbackSubmissionEdit so I replicated it so that it feels more standardized, should I revert it to use constants instead?